packages/castmill/lib/castmill/devices.ex

```diff
@@ -126,6 +147,35 @@ defmodule Castmill.Devices do
     count_devices(%{organization_id: nil, search: nil})
   end
 
+  @doc """
+    Mark a device as online. Sets the online field as well as the last_online field.
+  """
+  def mark_online(device_id, ip) do
+    fast_update_device(device_id, %{online: true, last_online: DateTime.utc_now(), last_ip: ip})
+  end
+
+  @doc """
+    Mark a device as offline. Sets the online field as well as the last_online field.
+  """
+  def mark_offline(device_id) do
+    fast_update_device(device_id, %{online: false, last_online: DateTime.utc_now()})
```

Comment:
so we set last_online to current time when marking a device as olffline. Is this the intended behavior?